api/assessment.go

```diff
@@ -159,11 +160,12 @@ type Assessment struct {
 	Stakeholders      []Ref                `json:"stakeholders"`
 	StakeholderGroups []Ref                `json:"stakeholderGroups" yaml:"stakeholderGroups"`
 	// read only
-	Risk         string                  `json:"risk"`
-	Confidence   int                     `json:"confidence"`
-	Status       string                  `json:"status"`
-	Thresholds   assessment.Thresholds   `json:"thresholds"`
-	RiskMessages assessment.RiskMessages `json:"riskMessages" yaml:"riskMessages"`
+	Risk                  string                  `json:"risk"`
+	Confidence            int                     `json:"confidence"`
+	Status                string                  `json:"status"`
+	Thresholds            assessment.Thresholds   `json:"thresholds"`
+	RiskMessages          assessment.RiskMessages `json:"riskMessages" yaml:"riskMessages"`
+	QuestionnaireRequired bool                    `json:"questionnaireRequired"`
```

Comment:
When you add a field with a camelCased name, be sure to add both a `json` and a `yaml` field tag with the camelCased spelling. (see the `RiskMessages` field  on the line above). This prevents a problem where the yaml encoder will emit it from the api as `camelcased` but the strict decoder will not recognize it because it expects `camelCased`.

nitpick: Lets just call this `Required` for simplicity and for symmetry with the other fields that are ultimately inherited from the questionnaire.